tox.ini

```diff
@@ -37,7 +37,7 @@ passenv =
     HOME
 commands =
     pip install -e .
-    pytest --cov=cookiecutter --cov-report=term --cov-report=html --cov-report=xml
+    pytest --cov=cookiecutter --cov-report=term --cov-report=html --cov-report=xml {posargs}
```

Comment:
why this change?

Reply:
This has been a dev leftover. I removed it now.